src/sass/components/_section--feedback.scss

```diff
@@ -0,0 +1,114 @@
+.section--feedback {
+  margin-bottom: 60px;
+
+  .panel-bar {
+    margin-bottom: 30px;
+
+    .row > * {
+      border-bottom: 4px solid #e2e2e2;
+    }
+
+    .heading {
+      position: relative;
+
+      h3 {
+        color: $primary;
```

Comment:
We shouldn't style by heading tags. Add className for this heading and use it here as selector.

Reply:
OK no problem :)

Reply:
done, but this first solution is used everywhere - in sections/components. shouldn't we change it?

Reply:
Yes, it would be a good thing :) We'll add new task for this.